### ISSUE TITLE
screen resolution

### ISSUE THREAD

**Issue by lggdev7** [balopez83/macOS_On_Hyper-V#3]
After several tests I have managed to start the virtual machine with Monterey.
But the resolution only allows me 1024x768.
Did you achieve another type of resolution?

Greetings and thanks.

**Comment by balopez83** [comment]
Hi @lggdev7 unfortunately this is an issue caused with the lack of graphics acceleration and the default resolution passed to the macOS virtual machine by Hyper-V.

The only way this might be resolved will be through the work of [acidanthera](https://github.com/acidanthera/MacHyperVSupport) through the development of special kexts.

**Comment by balopez83** [comment]
Can I trouble you to send me your config.plist. I have had some challenges getting Monterey to function reliably and so am curious how you changed your plist to make it work like its supposed to. Please remove any SMBIOS information prior to sending if you do.

**Comment by lggdev7** [comment]
What do you mean by smbios? 
to clear the PlatformInfo values?

**Comment by balopez83** [comment]
Yes, the serial information if you added it to the config.plist. If you never added it then you don't need to clear anything.

**Comment by lggdev7** [comment]
I send you one in Windows 11 and another in Windows 10 and the processors and graphics of the PC where Hyper-V is running
[envio.zip](https://github.com/balopez83/macOS_On_Hyper-V/files/9993288/envio.zip)

**Comment by balopez83** [comment]
@lggdev7 thank you for sending the config files.

**Comment by MSDarwish2000** [comment]
I've found a workaround which may help anyone reviewing this issue. 

I've used [BetterDisplay](https://github.com/waydabber/BetterDisplay) to create a virtual display with the desired resolution (My actual screen resolution). Then I've used AnyDesk to access the second screen as Hyper-V viewer only supports the main screen.

**Comment by cbugk** [comment]
I have recently used this [superuser answer](https://superuser.com/a/1716949) to set screen resolution of Ubuntu on Hyper-V. Resolution 1024x768 can be overwritten this way without any facility to change it on the go. I can test it today, I suppose.

**Comment by balopez83** [comment]
> 

@cbugk did you have any luck setting an alternate screen resolution using that superuser method? I tried multiple things including that but did not have any luck

**Comment by cbugk** [comment]
@balopez83 It must have worked, for Ubuntu of course. Can't say I remember to be honest. My memory generally serves me poorly, if at all.

**Comment by cguo2013** [comment]
anyone has lucky to solve this problem?
